src/extension/ScriptRunner.ts

```diff
@@ -187,19 +187,126 @@ export class ScriptRunner {
     })
   }
 
-  private postProblem(str: string): void {
-    const dataList = str.split('\n')
-    const diagnosticDataList = dataList[2].split(':')
-    const rowPosition = parseInt(diagnosticDataList[1]) - 1
-    const colStartPosition = parseInt(diagnosticDataList[2]) - 1
-    const startPosition = new Position(rowPosition, colStartPosition)
-    const colEndPosition = colStartPosition + dataList[3].length - 1
-    const endPosition = new Position(rowPosition, colEndPosition)
-    const range = new Range(startPosition, endPosition)
-    const errorMsg = diagnosticDataList[4]
-    const diagnostic = new Diagnostic(range, errorMsg)
-    const diagnostics: Diagnostic[] = [diagnostic]
-    const uri = Uri.parse(diagnosticDataList[0])
-    this.diagnosticCollection.set(uri, diagnostics)
+  private postProblems(confFile: string): void {
+    const ignoreFolderRegex = '{certora-logs,conf}'
+    const resourceErrorsFile = 'resource_errors.json'
+    vscode.workspace
+      .findFiles(resourceErrorsFile, ignoreFolderRegex, 1)
+      .then(data => {
+        workspace.fs.readFile(data[0]).then(data => {
+          const decoder = new TextDecoder()
+          const content = decoder.decode(data)
+          const resource_error = this.getResourceError(content)
+
+          const pathRegex = /((\\|\/)[a-z0-9\s_@\-^!#$%&+=.{}]+)+/i
+          const locationRegex = /\d+:\d+/g
+
+          resource_error.topics.forEach(topic => {
+            if (topic.messages.length > 0) {
+              topic.messages.forEach(message => {
+                const curMessage: string = message.message
+
+                const path = pathRegex.exec(curMessage)
+                const location = locationRegex.exec(curMessage)
+                const problemPath = this.getPath(path, confFile)
+
+                const descriptiveMessage = (topic.name + ':' + curMessage)
```

Comment:
**reasoning**
I thought this can be helpful for the user, because it tells them more information about what the problem might be.. you are right about them being able to figure it out without first categorizing the problem as type_error, Runtime solc etc, but I thought it adds value to know what type of problem it is. 
The only thing that bothers me about it is that the names are inconsistent in format, and that doesn't look good and also can be confusing maybe when you sometimes get a type error from solc and sometimes from type_error. I know this happens because the type error was once in the solidity file and them another time in the spec file, but for the user this might be confusing. Also there might be other confusing name I'm unaware about.
**current conclusion** 
I'll drop the names.